ui/app/templates/components/dashboard/quick-actions-card.hbs

```diff
@@ -0,0 +1,59 @@
+<h3 class="title is-4">Quick actions</h3>
+
+<div class="has-bottom-margin-m">
+  <h4 class="title is-6">Secrets engine</h4>
+  <SearchSelect
+    @id="secret-engines-select"
+    @options={{this.secretsEnginesOptions}}
+    @selectLimit="1"
+    @disallowNewItems={{true}}
+    @fallbackComponent="input-search"
+    @onChange={{this.handleSearchEngineSelect}}
+    @placeholder="Type to select a mount..."
+    @displayInherit={{true}}
+    @shouldRenderName={{true}}
+    @nameKey={{"name"}}
+    class="is-marginless"
+  />
+</div>
+
+<h4 class="title is-6">Action</h4>
+<Select
+  @name="auth-method"
+  @options={{this.selectedActions}}
+  @valueAttribute="actionType"
+  @labelAttribute="actionTitle"
+  @isFullwidth={{true}}
+  @selectedValue={{this.selectedAction}}
+  @onChange={{this.setSelectedAction}}
+/>
+
+<h4 class="title is-6">{{this.actionParamField.title}}</h4>
+{{#if (eq this.actionParamField.elementType "input")}}
+  <Input @type="text" @value={{this.value}} class="input" />
+{{else}}
+  <SearchSelect
+    class="is-flex-1"
+    @selectLimit="1"
+    @models={{array this.actionParamField.model}}
+    @backend={{this.selectedEngineName}}
+    @placeholder={{this.actionParamField.placeholder}}
+    @disallowNewItems={{true}}
+    @onChange={{this.handleSearch}}
+    @fallbackComponent="input-search"
+    @nameKey={{this.actionParamField.nameKey}}
+    @shouldRenderName={{true}}
```

Comment:
Does the fallbackComponent not handle the input state here? e.g. you have the conditional on row 32, but the idea with the fallbackComponent is it's shows the fallback if you can't do a searchSelect.

Reply:
The `fallbackComponent` does handle the input state. The conditional on line 32 is to check if the display is supposed to be an input. There are some quick actions (i.e. the kv action) that show an input box and not a SearchSelect box. Please let me know if that makes sense! :D